webpack.common.js

```diff
@@ -27,6 +27,9 @@ module.exports = {
   },
   resolve: {
     // .json is added to prevent import error from /node_modules/got/index.js
+    alias: {
+      'original-require': require.resolve('./polyfills/original-require'),
+    },
```

Comment:
This comment separates a comment from its target.

Reply:
Thank you @cds-amal! I'll fix that :)